src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -21,7 +21,10 @@
         "NETStandard.Library": {
           "version": "1.6.1",
           "exclude": "runtime"
-        }
+        },
+        "Microsoft.NETCore.Platforms": "1.1.0",
+        "Microsoft.NETCore.Runtime.CoreCLR": "1.1.0",
+        "Microsoft.NETCore.ILAsm": "2.0.0-beta-25027-03",
```

Comment:
Ah, expanded an "outdated" thread and see what you mean. I think it's just that one build so far that still had fail on stderr enabled, so I don't think finding another way's a high priority.